src/libaktualizr/uptane/fetcher_test.cc

```diff
@@ -86,7 +86,7 @@ TEST(fetcher, fetch_with_pause) {
     std::this_thread::sleep_for(std::chrono::milliseconds(200));
     EXPECT_EQ(f.setPause(true), PauseResult::kPaused);
     EXPECT_EQ(f.setPause(true), PauseResult::kAlreadyPaused);
-    std::this_thread::sleep_for(std::chrono::seconds(2));
+    std::this_thread::sleep_for(std::chrono::seconds(15));
```

Comment:
Why does it need to sleep so long?

Reply:
I just tried to understand who blocks the thread, I have already found it, but havn't restored previouse values